### ISSUE TITLE
Replace select2 with selectWoo

### ISSUE THREAD

**Issue by davetgreen** [kapow-wp/kapow-boilerplate#29]
After the recent announcement that the WooCommerce team have forked select2 due to a year without any real updates and the need for a11y improvements, we should be switching to use this instead of the original library.

See: https://woocommerce.wordpress.com/2017/08/08/selectwoo-an-accessible-replacement-for-select2/

**Comment by davetgreen** [comment]
Closing as select2 has seen more activity, and there is potential for PRs from selectwoo back into select2.
